Cython/Compiler/Code.py

```diff
@@ -1478,6 +1475,11 @@ def new_const_cname(self, prefix='', value=''):
             prefix = Naming.const_prefix
         return "%s%s" % (prefix, name_suffix)
 
+    def new_array_const_cname(self, prefix: str):
+        count = self.array_consts.get(prefix, 0)
+        self.array_consts[prefix] = count+1
+        return f"{Naming.pyrex_prefix}{prefix}[{count}]"
```

Comment:
I don't think this gives nice results if `prefix=''`. Is it still an optional argument?

Reply:
`prefix=''` was only used for function default arguments, which I've separated out.

I'll make it non-optional